doc/source/shell_completion.rst

```diff
@@ -1,34 +1,41 @@
-Shell auto-completion
-=====================
+Shell Completion
+================
 
-Papis has a bash auto-completion script that comes installed
-when you install papis with ``pip3``.
+Papis has shell completion for ``bash``, ``fish`` and ``zsh`` through the
+`click framework <https://click.palletsprojects.com/en/latest/shell-completion/#shell-completion>`__.
+that comes with it when installed through ``pip``.
 
-It should be installed in a relative path
+Normally, the completion should be installed to standard paths so that they
+are recognized by the shell. However, they can also be sourced manually:
 
-::
+* ``bash``: the completion script is installed in ``$PREFIX/etc/bash_completion.d``
+  and can be sourced manually using (or added to your ``.bashrc``)
 
-  PREFIX/etc/bash_completion.d/papis
+.. code:: bash
 
-normally the ``PREFIX`` part is ``/usr/local/``, so you can add the
-following line to your ``~/.bashrc`` file
+    source $PREFIX/etc/bash_completion.d/papis.sh
```

Comment:
I think, it could be better to install it in `/share/bash-completion/completions` because it's the path used by bash-completions, and it's supported by Linux distributions
```suggestion
    source $PREFIX/share/bash-completion/completions/papis
```

Reply:
Updated both here and in `setup.py`.
https://github.com/papis/papis/compare/25da9438ce1d38605295247de831d09939f0db98..b52d04d886cbf604c73ecc5457061d1017c6b85b